modules/placement-driver-api/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriver.java

```diff
@@ -60,6 +61,28 @@ CompletableFuture<ReplicaMeta> awaitPrimaryReplica(
             TimeUnit unit
     );
 
+    /**
+     * Temporary solution for awaiting {@link ReplicaMeta}. Waits for
```

Comment:
If it's temporary please add todo with a link and mark the method as @Deprecated. Please create a ticket if necessary.